msdata-sqlite/custom-sqlite.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Custom SQLite Versions - Microsoft.Data.Sqlite
+author: bricelam
+ms.date: 11/25/2019
+---
+# Custom SQLite Versions
+
+Microsoft.Data.Sqlite is built on top of SQLitePCL.raw. You can use custom versions of the native SQLite library be using a bundle or by configuring a SQLitePCL.raw provider.
+
+## Bundles
+
+SQLitePCL.raw provides bundle packages that make it easy to bring in the right dependencies across different platforms.
```

Comment:
FWIW, way back in my 1.0 release, I changed the nuget package IDs to start with "SQLitePCLRaw", but I kept the name of the GitHub repo as "SQLitePCL.raw" and also used that spelling in prose.  I grew to dislike that inconsistency, so, starting with my 2.0 release, I've been using the name "SQLitePCLRaw" in prose.  So at least now the prose name matches the nuget package IDs.